.github/workflows/test.yml

```diff
@@ -20,12 +20,10 @@ on:
     branches: main
 
 jobs:
-  test:
+  unit-test:
```

Comment:
Don't forget to go into the settings and change the required tests - this changes the CI job name and the required check goes by name.

Reply:
Yep, it's in the PR comments above. I won't forget!